xml/osc_building.xml

```diff
@@ -20,8 +20,120 @@
  
  <sect1 xml:id="sec.obs.building.spec2rpm">
   <title>From Spec Files (RPM)</title>
-  <para/>
-  <remark>TBD</remark>
+  <para>
+   To create an RPM package you need:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     the spec file
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     a file with the extension <filename class="extension">.changes</filename>
+    </para>
+   </listitem>
+   <listitem>
+    <para>the original source archive</para>
+   </listitem>
+   <listitem>
+    <para>
+     optional patches
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     other files which does not fall into one of the previous categories
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   For existing packages, this is already the case. To build an
+   existing package, the general procedure is as follows:</para>
+  <procedure>
+   <step>
+    <para>
+     If you have not done yet, set up your project as shown in
+     <xref linkend="sec.obs.basicworkflow.setuphome"/>.
+    </para>
+   </step>
+   <step>
+    <para>
+     In the terminal, choose or create a directory on a local partition that
+     has enough space to hold the package sources.
+    </para>
+   </step>
+   <step>
+    <para>Check out the project where the package is contained:</para>
```

Comment:
better: "Check out the project that contains the package:"

Reply:
Right, fixed.